integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/blockprune/BlockPruneQueryTestCase.scala

```diff
@@ -18,16 +18,30 @@ package org.apache.carbondata.spark.testsuite.blockprune
 
 import java.io.DataOutputStream
 
+import org.apache.carbondata.core.constants.CarbonCommonConstants
 import org.apache.spark.sql.Row
 import org.scalatest.BeforeAndAfterAll
 import org.apache.carbondata.core.datastore.impl.FileFactory
+import org.apache.carbondata.core.util.CarbonProperties
 import org.apache.spark.sql.test.util.QueryTest
 
 /**
   * This class contains test cases for block prune query
   */
 class BlockPruneQueryTestCase extends QueryTest with BeforeAndAfterAll {
   val outputPath = s"$resourcesPath/block_prune_test.csv"
+  val MULTI_THREAD_ENABLE_FILES_COUNT = "1";
+  val MULTI_THREAD_DISABLE_FILES_COUNT
+    = CarbonCommonConstants.CARBON_DRIVER_PRUNING_MULTI_THREAD_ENABLE_FILES_COUNT_DEFAULT;
+
+  def perpareCarbonProperty(propertyName:String,
+    propertyValue:String): Unit ={
+    val properties = CarbonProperties.getInstance()
+    properties.removeProperty(propertyName)
```

Comment:
removeProperty may not be required, as addProperty in next line will update the key with new value if already present

Reply:
resolved